2024-08-07 18:37:18,142 INFO    StreamThr :31767 [internal.py:wandb_internal():86] W&B internal server running at pid: 31767, started at: 2024-08-07 18:37:18.142323
2024-08-07 18:37:18,143 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: status
2024-08-07 18:37:18,144 INFO    WriterThread:31767 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/run-glkydmt7.wandb
2024-08-07 18:37:18,144 DEBUG   SenderThread:31767 [sender.py:send():382] send: header
2024-08-07 18:37:18,177 DEBUG   SenderThread:31767 [sender.py:send():382] send: run
2024-08-07 18:37:18,343 INFO    SenderThread:31767 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files
2024-08-07 18:37:18,343 INFO    SenderThread:31767 [sender.py:_start_run_threads():1136] run started: glkydmt7 with start time 1723027038.142278
2024-08-07 18:37:18,348 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: check_version
2024-08-07 18:37:18,348 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: check_version
2024-08-07 18:37:18,439 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: run_start
2024-08-07 18:37:18,449 DEBUG   HandlerThread:31767 [system_info.py:__init__():32] System info init
2024-08-07 18:37:18,449 DEBUG   HandlerThread:31767 [system_info.py:__init__():47] System info init done
2024-08-07 18:37:18,449 INFO    HandlerThread:31767 [system_monitor.py:start():194] Starting system monitor
2024-08-07 18:37:18,449 INFO    SystemMonitor:31767 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-07 18:37:18,449 INFO    SystemMonitor:31767 [interfaces.py:start():190] Started cpu monitoring
2024-08-07 18:37:18,449 INFO    HandlerThread:31767 [system_monitor.py:probe():214] Collecting system info
2024-08-07 18:37:18,450 INFO    SystemMonitor:31767 [interfaces.py:start():190] Started disk monitoring
2024-08-07 18:37:18,450 INFO    SystemMonitor:31767 [interfaces.py:start():190] Started gpu monitoring
2024-08-07 18:37:18,451 INFO    SystemMonitor:31767 [interfaces.py:start():190] Started memory monitoring
2024-08-07 18:37:18,451 INFO    SystemMonitor:31767 [interfaces.py:start():190] Started network monitoring
2024-08-07 18:37:18,457 DEBUG   HandlerThread:31767 [system_info.py:probe():196] Probing system
2024-08-07 18:37:18,459 DEBUG   HandlerThread:31767 [system_info.py:_probe_git():181] Probing git
2024-08-07 18:37:18,461 DEBUG   HandlerThread:31767 [system_info.py:_probe_git():189] Probing git done
2024-08-07 18:37:18,461 DEBUG   HandlerThread:31767 [system_info.py:probe():244] Probing system done
2024-08-07 18:37:18,461 DEBUG   HandlerThread:31767 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-07T10:37:18.457899', 'startedAt': '2024-08-07T10:37:18.137167', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/llama2-7b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '1', '--gradient_accumulation_steps', '32', '--save_strategy', 'steps', '--save_steps', '4000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 2316.3858214285715, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 3804.502, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2536.844, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2053.801, 'min': 800.0, 'max': 5500.0}, {'current': 2601.36, 'min': 800.0, 'max': 5500.0}, {'current': 2292.298, 'min': 800.0, 'max': 5500.0}, {'current': 3170.933, 'min': 800.0, 'max': 5500.0}, {'current': 2277.395, 'min': 800.0, 'max': 5600.0}, {'current': 1260.157, 'min': 800.0, 'max': 5600.0}, {'current': 2607.222, 'min': 800.0, 'max': 5600.0}, {'current': 4628.045, 'min': 800.0, 'max': 5600.0}, {'current': 2610.431, 'min': 800.0, 'max': 5500.0}, {'current': 2225.636, 'min': 800.0, 'max': 5500.0}, {'current': 3129.466, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 3832.461, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 474.68243408203125}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-07 18:37:18,461 INFO    HandlerThread:31767 [system_monitor.py:probe():224] Finished collecting system info
2024-08-07 18:37:18,461 INFO    HandlerThread:31767 [system_monitor.py:probe():227] Publishing system info
2024-08-07 18:37:18,462 DEBUG   HandlerThread:31767 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-07 18:37:18,462 DEBUG   HandlerThread:31767 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-07 18:37:18,462 DEBUG   HandlerThread:31767 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-07 18:37:19,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/conda-environment.yaml
2024-08-07 18:37:19,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/requirements.txt
2024-08-07 18:37:20,147 DEBUG   HandlerThread:31767 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-07 18:37:20,147 DEBUG   HandlerThread:31767 [system_info.py:_save_code():90] Saving code
2024-08-07 18:37:20,149 DEBUG   HandlerThread:31767 [system_info.py:_save_code():111] Saving code done
2024-08-07 18:37:20,149 DEBUG   HandlerThread:31767 [system_info.py:_save_patches():128] Saving git patches
2024-08-07 18:37:20,188 DEBUG   HandlerThread:31767 [system_info.py:_save_patches():170] Saving git patches done
2024-08-07 18:37:20,189 INFO    HandlerThread:31767 [system_monitor.py:probe():229] Finished publishing system info
2024-08-07 18:37:20,191 DEBUG   SenderThread:31767 [sender.py:send():382] send: files
2024-08-07 18:37:20,191 INFO    SenderThread:31767 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-07 18:37:20,192 INFO    SenderThread:31767 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-07 18:37:20,192 INFO    SenderThread:31767 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-07 18:37:20,200 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: stop_status
2024-08-07 18:37:20,202 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: stop_status
2024-08-07 18:37:20,207 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-07 18:37:20,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/conda-environment.yaml
2024-08-07 18:37:20,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/diff.patch
2024-08-07 18:37:20,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/code/llama/gmm.py
2024-08-07 18:37:20,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/wandb-metadata.json
2024-08-07 18:37:20,343 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/code
2024-08-07 18:37:20,344 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/code/llama
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: telemetry
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: config
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: metric
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: telemetry
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: metric
2024-08-07 18:37:20,391 WARNING SenderThread:31767 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: telemetry
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: telemetry
2024-08-07 18:37:20,391 DEBUG   SenderThread:31767 [sender.py:send():382] send: config
2024-08-07 18:37:20,438 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea
2024-08-07 18:37:20,438 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5224', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:20,438 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96BF6DBBE5D94</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:20,475 DEBUG   SenderThread:31767 [sender.py:send():382] send: exit
2024-08-07 18:37:20,475 INFO    SenderThread:31767 [sender.py:send_exit():589] handling exit code: 1
2024-08-07 18:37:20,475 INFO    SenderThread:31767 [sender.py:send_exit():591] handling runtime: 2
2024-08-07 18:37:20,476 INFO    SenderThread:31767 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:37:20,476 INFO    SenderThread:31767 [sender.py:send_exit():597] send defer
2024-08-07 18:37:20,476 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,476 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-07 18:37:20,476 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,476 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-07 18:37:20,476 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 1
2024-08-07 18:37:20,476 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,476 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-07 18:37:20,476 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,476 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-07 18:37:20,476 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 2
2024-08-07 18:37:20,476 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,476 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-07 18:37:20,476 INFO    HandlerThread:31767 [system_monitor.py:finish():203] Stopping system monitor
2024-08-07 18:37:20,476 INFO    HandlerThread:31767 [interfaces.py:finish():202] Joined cpu monitor
2024-08-07 18:37:20,477 INFO    HandlerThread:31767 [interfaces.py:finish():202] Joined disk monitor
2024-08-07 18:37:20,477 DEBUG   SystemMonitor:31767 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-07 18:37:20,477 DEBUG   SystemMonitor:31767 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-07 18:37:20,477 DEBUG   SystemMonitor:31767 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-07 18:37:20,496 INFO    HandlerThread:31767 [interfaces.py:finish():202] Joined gpu monitor
2024-08-07 18:37:20,496 INFO    HandlerThread:31767 [interfaces.py:finish():202] Joined memory monitor
2024-08-07 18:37:20,496 INFO    HandlerThread:31767 [interfaces.py:finish():202] Joined network monitor
2024-08-07 18:37:20,496 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,496 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-07 18:37:20,496 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 3
2024-08-07 18:37:20,496 DEBUG   SenderThread:31767 [sender.py:send():382] send: stats
2024-08-07 18:37:20,496 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,496 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-07 18:37:20,496 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,496 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-07 18:37:20,496 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 4
2024-08-07 18:37:20,497 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,497 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-07 18:37:20,497 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,497 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-07 18:37:20,497 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 5
2024-08-07 18:37:20,497 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,497 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-07 18:37:20,497 DEBUG   SenderThread:31767 [sender.py:send():382] send: summary
2024-08-07 18:37:20,497 INFO    SenderThread:31767 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:37:20,497 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,497 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-07 18:37:20,497 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 6
2024-08-07 18:37:20,497 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,497 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-07 18:37:20,497 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,497 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-07 18:37:20,499 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: status_report
2024-08-07 18:37:20,558 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a
2024-08-07 18:37:20,558 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:20,559 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96BF6DE725E09</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:20,610 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 7
2024-08-07 18:37:20,610 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:20,610 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-07 18:37:20,610 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:20,610 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-07 18:37:20,899 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31
2024-08-07 18:37:20,899 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '289653', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:20,899 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96BF6E3B05B59</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:21,344 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/config.yaml
2024-08-07 18:37:21,345 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/output.log
2024-08-07 18:37:21,345 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/wandb-summary.json
2024-08-07 18:37:21,475 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:37:21,529 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea
2024-08-07 18:37:21,529 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5224', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:21,529 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96BF71CE35F02</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:21,800 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a
2024-08-07 18:37:21,800 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:21,800 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96BF72CF0397A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:21,800 INFO    wandb-upload_1:31767 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:37:22,336 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31
2024-08-07 18:37:22,336 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '289653', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:22,336 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96BF7428E716D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:22,413 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 8
2024-08-07 18:37:22,413 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: poll_exit
2024-08-07 18:37:22,413 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:22,413 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-07 18:37:22,413 DEBUG   SenderThread:31767 [sender.py:send_request():409] send_request: defer
2024-08-07 18:37:22,413 INFO    SenderThread:31767 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-07 18:37:22,413 INFO    SenderThread:31767 [job_builder.py:build():298] Attempting to build job artifact
2024-08-07 18:37:22,414 INFO    SenderThread:31767 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-07 18:37:22,428 INFO    SenderThread:31767 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-07 18:37:22,440 INFO    SenderThread:31767 [sender.py:transition_state():617] send defer: 9
2024-08-07 18:37:22,441 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:37:22,441 INFO    HandlerThread:31767 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-07 18:37:22,441 DEBUG   SenderThread:31767 [sender.py:send():382] send: artifact
2024-08-07 18:37:22,475 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:37:23,027 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61
2024-08-07 18:37:23,027 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:23,027 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96BF77637D71E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:23,076 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2
2024-08-07 18:37:23,076 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:23,076 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96BF77676DCFC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:23,345 INFO    Thread-12 :31767 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_183718-glkydmt7/files/output.log
2024-08-07 18:37:23,863 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea
2024-08-07 18:37:23,863 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5224', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:23,863 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96BF7A80C93F4</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:23,942 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a
2024-08-07 18:37:23,942 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:23,942 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96BF7ACACC7F1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:24,117 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61
2024-08-07 18:37:24,118 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:24,118 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96BF7B733E9EF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:24,318 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2
2024-08-07 18:37:24,319 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:24,319 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96BF7C1F96EB4</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:24,800 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31
2024-08-07 18:37:24,800 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '289653', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:24,801 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96BF7D779C748</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:26,320 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61
2024-08-07 18:37:26,321 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:26,321 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96BF83A611450</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:26,321 INFO    wandb-upload_3:31767 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:37:26,567 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2
2024-08-07 18:37:26,567 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:26,567 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96BF8482C71DD</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:26,567 INFO    wandb-upload_4:31767 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:37:27,478 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:28,001 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea
2024-08-07 18:37:28,001 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5224', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:28,001 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96BF89EA47A71</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:28,975 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a
2024-08-07 18:37:28,975 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:28,976 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96BF8D8978DC6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:29,382 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31
2024-08-07 18:37:29,382 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '289653', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:29,382 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96BF8E8FB9692</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:30,958 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2
2024-08-07 18:37:30,958 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:30,959 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96BF94DF6FB52</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:31,174 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61
2024-08-07 18:37:31,175 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:31,175 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96BF95BABCB71</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:32,481 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:37,182 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea
2024-08-07 18:37:37,182 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5224', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:37,182 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96BFAC1EC6614</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:37,483 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:37,802 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a
2024-08-07 18:37:37,802 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:37,802 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96BFAE6B166B8</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:38,819 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31
2024-08-07 18:37:38,819 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '289653', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:38,819 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96BFB06432CAD</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:39,937 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61
2024-08-07 18:37:39,938 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:39,938 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96BFB6602833B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:40,774 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2
2024-08-07 18:37:40,774 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:40,774 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96BFB9712144C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:42,486 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:47,489 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:52,491 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:55,591 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=72ca1fe424ad05e16285557c316b38a2f72b752b843c3add92efcf524d9e00ea
2024-08-07 18:37:55,591 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5224', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:55,591 ERROR   wandb-upload_0:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96BFF0B14D464</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:55,743 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=37b27c23aeaeb4cadb93635d304bdf707dc27be8f3c650e5beb0de1a69aa805a
2024-08-07 18:37:55,743 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:55,743 ERROR   wandb-upload_1:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96BFF0FF11452</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:57,494 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:37:58,080 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/glkydmt7/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103720Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a7acaeffaf3a703f872bfcddffa57cda340fe441a0702e3cd1e5cc107d51ac31
2024-08-07 18:37:58,081 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '289653', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:58,081 ERROR   wandb-upload_2:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/glkydmt7/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/glkydmt7/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96BFF7E3151C0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:37:58,783 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f671fc537c5f6318d8d636dc4455e681405f28e9897edf6c2a1ad48007c591b2
2024-08-07 18:37:58,783 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:37:58,783 ERROR   wandb-upload_4:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96BFFC684862F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:38:00,068 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T103722Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=8bdba1917f4f29fe6d80e72bb23217de678a2e2ca792558ac7ea8fa7bd3f3e61
2024-08-07 18:38:00,068 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:38:00,068 ERROR   wandb-upload_3:31767 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/515/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96C00161CF21C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:38:02,497 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:38:07,500 DEBUG   HandlerThread:31767 [handler.py:handle_request():146] handle_request: keepalive
